src/composables/useCamera/index.ts

```diff
@@ -23,15 +24,21 @@ export const useCamera = ({ sizes }: Pick<TresContext, 'sizes'> & { scene: TresS
     cameras.value = [camera, ...otherCameras]
   }
 
-  const registerCamera = (newCamera: Camera, active = false) => {
-    if (cameras.value.some(({ uuid }) => uuid === newCamera.uuid)) { return }
+  const registerCamera = (maybeCamera: any, active = false) => {
```

Comment:
Here seems we are replacing a Type guard `newCamera: Camera` with a `isCamera(maybeCamera)`

Setting `any` is the same as not using Typescript at all, please use `unknown` type instead in all places where you don't know exactly what type to recieve

Reply:
Sounds good. Will change.